### PR TITLE
Run DWD lookup before KGTK for x-variables and claimers

### LINKED ISSUE #46: Query DWD first for QNodes before querying KGTK  (`XVariable` & `Claimer`)
Our primary resource for QNode linking should be the DWD set of events. If we don't find anything, then we should go to the KGTK.

### PR DESCRIPTION
**Pull request by elizlee**
Closes #46

### CODE REVIEW

File: cdse_covid/semantic_extraction/run_wikidata_linking.py [hunk 1]
```diff
@@ -15,6 +18,59 @@ def _find_links(span: str, tokens: Sequence[str]) -> Any:
     return (disambiguate_kgtk(span, token, k=1) for token in tokens)
 
 
+def get_best_qnode_for_xvariable_or_claimer(
```
Comment (joecummings) [inline cdse_covid/semantic_extraction/run_wikidata_linking.py:21]: Why is this specific to XVariable/Claimer? Couldn't this theoretically be used to get the best Qnode for any given string?

File: wikidata_linker/disambiguate_with_amr.py [hunk 2]
```diff
@@ -126,44 +125,44 @@ def get_wikidata_for_labeled_args(
...
     # Load both qnode mappings
     with open(master_table_path, "r", encoding="utf-8") as in_json:
         pbs_to_qnodes_master = json.load(in_json)
     with open(overlay_table_path, "r", encoding="utf-8") as in_json_2:
         pbs_to_qnodes_overlay = json.load(in_json_2)
 
+    return pbs_to_qnodes_master, pbs_to_qnodes_overlay
+
+
+def disambiguate_with_amr(
```
Comment (joecummings) [inline wikidata_linker/disambiguate_with_amr.py:148]: This should probably get changed to something like `get_claim_semantics_with_amr`.

**Comment by joecummings** [comment]
`run_dwd_overlay` should probably be changed to something that directly references what it is doing, which is getting claim semantics. Either that OR this code could be put in the `run_amr` file. There'd be less parallelization but it would logically make sense. Thoughts on either solution?

At some point, I'd personally like to separate the methods from the products. For example, having a `find_x_variable` job that can use SRL or AMR under the hood.

File: cdse_covid/semantic_extraction/run_wikidata_linking.py (hunk 1 above)
Comment (elizlee) [reply cdse_covid/semantic_extraction/run_wikidata_linking.py:21]: It could, yes

**Comment by joecummings** [comment]
Also, can you make sure you're running the `precommit` checks? I think I'm getting some residual errors from `disambiguate_with_amr`.

**Comment by elizlee** [comment]
> run_dwd_overlay should probably be changed to something that directly references what it is doing, which is getting claim semantics. Either that OR this code could be put in the run_amr file. There'd be less parallelization but it would logically make sense. Thoughts on either solution?

I agree. We're getting to a point where such reorganization would be good.

File: wikidata_linker/get_claim_semantics.py [hunk 3]
```diff
@@ -11,15 +11,19 @@
...
 from wikidata_linker.wikidata_linking import disambiguate_kgtk
 
 OVERLAY = "overlay"
 MASTER = "master"
 
 PARENT_DIR = Path(__file__).parent
+ORIGINAL_MASTER_TABLE = PARENT_DIR / "resources" / "qe_master.json"
 
-FRAME_LABEL_PATTERN = r"[a-z-]+-[0-9]{2}"  # e.g. have-name-91
+STOP_WORDS = ["and", "or", "the", "a", "is", "are", "like", "in"]
```
Comment (joecummings) [inline wikidata_linker/get_claim_semantics.py:26]: Why these ones and not the list from NLTK?

Comment (joecummings) [reply wikidata_linker/get_claim_semantics.py:26]: Also, at the very least can convert this to a set with `{}`

Comment (elizlee) [reply wikidata_linker/get_claim_semantics.py:26]: 1. I forgot NLTK had such a list
2. "like" appears often in the test data and isn't in the list, but it looks easy enough to add it myself.

Comment (elizlee) [reply wikidata_linker/get_claim_semantics.py:26]: I had changed it to a list earlier to appease the precommit checks, though it doesn't seem to have a problem with this new NLTK stopword set, so I'm not sure what the problem was...

**Review by joecummings — APPROVED** [review]
LGTM

**Pull request merged**